LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -2617,6 +2617,43 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
                         c.Add(new Control("Chipset Fan", 3));
                         break;
 
+                    case Model.Z690_Extreme:
+                        v.Add(new Voltage("Vcore", 0, 10, 10));
+                        v.Add(new Voltage("+5V", 1, 20, 10));
+                        v.Add(new Voltage("AVCC", 2, 34, 34));
+                        v.Add(new Voltage("+3.3V", 3, 34, 34));
+                        v.Add(new Voltage("+12V", 4, 110, 10));
+                        v.Add(new Voltage("CPU Input Voltage", 5, 1, 1));
+                        v.Add(new Voltage("DRAM", 6));
+                        v.Add(new Voltage("+3.3V Standby", 7, 34, 34));
+                        v.Add(new Voltage("VBat", 8, 34, 34));
```

Comment:
Although I kept _VBat_ for consistency naming reasons, I would have felt that **CMOS Battery Voltage** would have been more appropriate. Let me know if I should keep as is or alter the name

Reply:
"CMOS Battery" makes sense over VBat, same here with the other names in this.